### PR TITLE
Add parents to orphaned terms

### PR DESCRIPTION
**Pull request by shawntanzk**
Fixes https://github.com/obophenotype/uberon/issues/2987

Did not do much digging/research, just adding basic parents, more precise parentage can come in another PR if needed

### CODE REVIEW

File: src/ontology/uberon-edit.obo [hunk 1]
```diff
@@ -128290,6 +128290,7 @@ xref: NCIT:C33426
 xref: SCTID:361365004
 xref: UMLS:C0225696 {source="ncithesaurus:Pulmonary_Acinus"}
 xref: Wikipedia:Acinus#The_Lungs
+is_a: UBERON:0034922 ! cell cluster
```
Comment (aleixpuigb) [inline src/ontology/uberon-edit.obo:128293]: The pulmonary acinus contain alveolar ducts and respiratory bronchioles, I don't think it should be classified as a cell cluster. Should it also be an anatomical entity?

**Review by aleixpuigb — CHANGES_REQUESTED** [review]
Please, consider a different parent for pulmonary acinus.

File: src/ontology/uberon-edit.obo (hunk 1 above)
Comment (shawntanzk) [reply src/ontology/uberon-edit.obo:128293]: happy to put it as anatomical entity :) only put cell cluster cause I searched up acinus and it said cell cluster.

Comment (aleixpuigb) [reply src/ontology/uberon-edit.obo:128293]: Yes, the pulmonary acinus is a bit different than other acini.

**Comment by aleixpuigb** [comment]
I have received a notification of this commit [14dd3a6](https://github.com/obophenotype/uberon/commit/14dd3a618a7ddc9c38c11d8658eb7a175acf33a7), but I can't see it in the PR. Something similar happened to #2983. I can see the commit in the branch, but it will not merge if you mergw the PR.
image
Any idea why does that happen?

**Comment by shawntanzk** [comment]
ok a bit confused, hate not having github desktop, but I have committed the requested change to the branch (https://github.com/obophenotype/uberon/commit/14dd3a618a7ddc9c38c11d8658eb7a175acf33a7) but it does not seem to be showing up in the PR - Is there something I am doing wrong?

**Comment by shawntanzk** [comment]
ok fixed, sync issue apparently, see https://stackoverflow.com/questions/45626986/updating-pull-request-after-review-changes-commit-doesnt-show-up-in-the-pull-re

**Review by aleixpuigb — APPROVED** [review]

**Review by gouttegd — APPROVED** [review]

**Pull request merged**